ssv/types.go

```diff
@@ -32,7 +32,7 @@ type AttesterCalls interface {
 	GetAttestationData(slot *phase0.Slot, committeeIndex *phase0.CommitteeIndex) (*phase0.AttestationData,
```

Comment:
not part of this PR but since you are editing this already, I don't know why you added pointers to slot and committeeIndex, it should be like this -
`slot phase0.Slot, committeeIndex phase0.CommitteeIndex` as these are int values

Reply:
you are wrong, they are required 
![image](https://github.com/ssvlabs/ssv-spec/assets/2600603/a0ec452b-ee2c-4346-8422-2c6761be51c6)

please revert back to using values instead of pointers.

Reply:
hmm I was looking in the `Beacon` api and not `Validator` like you did.
But I think you are correct, the `Validator` one should be used

But what happens when a committee runner asks validators to create attestations for the wrong committee index?
Are we sure that it works on all clients?
If it does, I am not sure that we have a promise that later on this won't break by some validation?

Reply:
we are using the validator api in all clients already.
not sure about your first question though.